app/models/person.rb

```diff
@@ -22,8 +22,19 @@ def title
     @content_item.content_item_data["title"]
   end
 
-  def roles
-    links["ordered_current_appointments"].map { |ra| ra["title"] }.to_sentence
+  def current_roles
+    appointments = links["ordered_current_appointments"]
+    return unless appointments
+
+    if appointments.include?(%w[links])
+      roles = appointments.map { |ra| ra["links"]["role"] }.map do |role|
+        role.map { |r| r["title"] }.join(",")
+      end
+
+      roles.to_sentence
+    else
+      appointments.map { |ra| ra["title"] }.map { |t| t.split(" - ").last }.to_sentence
```

Comment:
This should never be the case - is there an example content item that is missing the links?

Reply:
https://www.gov.uk/api/content/government/people/aarti-thakor

Reply:
Thanks. I'll investigate why this is the case. In the mean time you don't need to worry about supporting that kind of content item, it's incorrect.

It might also be worth adding a test with an example correct content item.